markets/perps-market/test/integration/bootstrap/bootstrapPerpsMarkets.ts

```diff
@@ -126,6 +126,13 @@ export const bootstrapPerpsMarkets = (
         );
       });
 
+      before('set max market value', async () => {
+        await contracts.PerpsMarket.connect(marketOwner).setMaxMarketValue(
+          marketId,
+          bn(10_000_000)
```

Comment:
dont hardcode this for every market.  create an optional parameter that gets passed in here.  if it doesn't exist then default to this value